docs/license.rst

```diff
@@ -7,5 +7,5 @@ in the ``qsirecon`` distribution.
 
 All trademarks referenced herein are property of their respective holders.
 
-Copyright (c) 2018-2019 PennLINC
+Copyright (c) 2018-2024 PennLINC
```

Comment:
I recall something about not needing to change end-dates for licenses. I think you can just have `2018`.